repositories/analytics_views/users.sql

```diff
@@ -0,0 +1,15 @@
+DROP VIEW IF EXISTS analytics.users;
+
+CREATE VIEW
+      analytics.users
+WITH
+      (security_invoker = false) AS
```

Comment:
For my curiosity, why do we need this ?

Reply:
So there are two ways of running a view:
- with the permissions of the user running the query
- with the permissions of the user that created the query

think of it like this:
- I have two tables A and B, and a user analytics that has permissions to read table A only.
- You define a view that reads from tables A and B
- if the view is defined with security_invoker=true, you won't be able to read (cuz the user can't read B) and if you set security_invoker=false, you will be able to read because the view is "its own black box" as long as the user that created it has access to A and B (assuming your user has select permissions on that view)